accord-core/src/test/java/accord/impl/basic/Cluster.java

```diff
@@ -190,6 +172,12 @@ private void processNext(Object next)
         }
     }
 
+    public void notifyDropped(Node.Id from, Node.Id to, long id, Message message)
```

Comment:
only here to keep the trace logging from before... im cool dropping.  We now have more detailed debug logs for messaging